primitives/src/sequence.rs

```diff
@@ -16,17 +16,17 @@
 //! [BIP-125]: <https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki>
 
 use core::fmt;
-
 use internals::impl_to_hex_from_lower_hex;
 #[cfg(feature = "serde")]
 use serde::{Deserialize, Serialize};
 #[cfg(feature = "alloc")]
 use units::locktime::relative::TimeOverflowError;
 #[cfg(feature = "alloc")]
 use units::parse::{self, PrefixedHexError, UnprefixedHexError};
-
 #[cfg(feature = "alloc")]
 use crate::locktime::relative;
+#[cfg(feature = "arbitrary")]
+use arbitrary::{Arbitrary, Unstructured};
```

Comment:
The weekly formatting job will fix this up but just so you know these should be in the section above.